NebulaPatcher/Patches/Transpilers/MechaDroneLogic_Patch.cs

```diff
@@ -68,8 +68,8 @@ static IEnumerable<CodeInstruction> UpdateTargets_Transpiler(IEnumerable<CodeIns
         /*
          * Call DroneManager.BroadcastDroneOrder(int droneId, int entityId, int stage) when drone's stage changes
          */
-        //[HarmonyTranspiler]
-        //[HarmonyPatch("UpdateDrones")]
+        [HarmonyTranspiler]
+        [HarmonyPatch("UpdateDrones")]
```

Comment:
What behavior is this supposed to fix? I believe I have updated this correctly but I'm unsure what I need to test to verify